shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/segment/dml/expr/FunctionSegment.java

```diff
@@ -40,4 +41,7 @@
     private final Collection<ExpressionSegment> parameters = new LinkedList<>();
     
     private final String text;
+    
+    @Setter
+    private String owner;
```

Comment:
Those existing FunctionName instances do not need these field, so I made it as optional field